python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -87,3 +95,55 @@ cpdef read_parquet(filepath_or_buffer, columns=None, row_group=None,
         df.index.name = new_index_name
 
     return df
+
+cpdef write_parquet(
+        _Table table,
+        path,
+        compression=None,
+        statistics="ROWGROUP"):
+    """
+    Cython function to call into libcudf API, see `write_parquet`.
+
+    See Also
+    --------
+    cudf.io.parquet.write_parquet
+    """
+
+    # Create the write options
+    cdef string filepath = <string>str(path).encode()
+    cdef sink_info sink = sink_info(filepath)
+    cdef table_metadata *tbl_meta = new table_metadata()
```

Comment:
Should we just use a `make_unique` here instead?

Reply:
I started to do that but "write_parquet_args" constructor does not accept a unique_ptr

Reply:
You can get the raw pointer from a unique_ptr using `.get()`

Reply:
Sure. I just didn't see the point in making a smart pointer to immediately not use it but I'll change to that since you are right that is what we do most of the time.

Reply:
Registering the smart pointer gives it a default deleter so it automatically calls `delete` on it when it goes out of scope instead of us manually deleting it at the end of the function. Basically just a safer practice.